themes/rad/layouts/index.html

```diff
@@ -4,25 +4,24 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Radicle a decentralized, collaborative revision control system built on IPFS.
+        Write open source software without giving your data to third parties or setting up a server.
       </p>
     </div>
     <div id="term">
       <h2>Terminal-first</h2>
       <p>
-        A Radicle project contains a git repository as well as the
-        issues and proposals associated with that repo. All accessible from
-        your terminal.
+        A Radicle project contains a git repository, plus the associated
+        issues and proposals. Access all the conversations
+        that happen around your codebase from your command line.
```

Comment:
"Conversation" sounds too loose and metaphorical for me. The takehome from the user testing sessions to me was that we should be as concrete as possible.

Reply:
(Also makes it seem like we have chat or email-like functionality.)